### DISCUSSION TITLE
Species Tree

### DISCUSSION THREAD

**Discussion by Rocket-Joe** [MaxRobinsonTheGreat/EvolutionSimulatorV2#106] [Ideas and Suggestions]
I think it would be interesting to be able to see the evolution tree of my simulation. You would be able to view all the past organisms and see the path that evolution took, which organisms died, which survived, and see how far back/recently different species emerged. This is a feature I really enjoy in other evolution games like _Species_ on Steam.

**Comment by theomgdev** [comment]
In fact, the codes are very suitable for this, the fossil record is already kept and the species are counted, a good interface for the fossil record can be created with appropriate classification criteria.
